### ISSUE TITLE
[Windows][Installation] Install of Ray using Windows subsystem for Linux (WSL) fails to start the Ray dashboard

### ISSUE THREAD

**Issue by ChristosPeridis** [ray-project/ray#32693]
Hello dear members of the Ray team.

I am a ML engineer with specialty in DRL. I am using Ray for accelerating my workflows and running multiple experiments by leveraging the Tune and Rllib Ray APIs. I have used Ray in Linux and Windows. For the development phase I usually run my code in my personal Windows machines for testing. Lately I have been using WSL for running my code on Windows11 since I needed Fork method for process initialization in my programme, which is not  natively supported by Windows. I wanted also to run Ray through WSL. However when I tried I faced the following error:

![image](https://user-images.githubusercontent.com/34765945/220319192-01eab147-39b9-4d45-9eb5-971d88a39f71.png)

Is there an easy way to overcome this issue, so I am able to run Ray using WSL? It is very important because certain ML libraries, such as [TensorFlow](https://www.tensorflow.org/install/pip), will stop offering their lates versions for native Windows, and the latest versions will only work through WSL. 

![image](https://user-images.githubusercontent.com/34765945/220321386-5dad471d-319e-4e0d-b745-cdf6e2211e7a.png)

WSL seems to be the future for ML application development on Windows, and Ray being able to support it by default will significantly increase the frameworks reputation.

I am always at your disposal for any further information or queries you might have for my set up of Ray or the error message.

Thank you very much for your valuable help!!!

Kind regards,

Christos Peridis

**Comment by sbarman-mi9** [comment]
Hi Christos,

See if the dashboard server is running at: localhost:8265

I too get the 'failed to start dashboard error' but when I go to localhosst:8265, it works fine. The logs also show that the dashboard server is running. Yet to figure why it spits out that error message.

![image](https://user-images.githubusercontent.com/14152255/222479285-a47b6984-6e77-4fa8-8c73-ce25fcc4b8e1.png)

![image](https://user-images.githubusercontent.com/14152255/222480557-c8eac6a3-1bae-4f0d-aae4-b299b76a2253.png)

<u>System Specs</u>:
Ubuntu 22.04 LTS
Windows 10 WSL2

**Comment by scottsun94** [comment]
cc: @rkooo567

**Comment by rkooo567** [comment]
What's your ray version? Can you try using Ray 2.3 and see what's the error message?

**Comment by ChristosPeridis** [comment]
Thank you very much everyone for your replies!

I do apologise for my delayed reply, unfortunately I was very busy this period with other running projects I did not manage to find time for further investigating to this issue. I had been using Ray version 2.2.0 when I first experimented with running Ray through WSL2. I will test it with Ray version 2.3.1 as well and I will report back to you as soon as possible.

Thank you all very much again for your valuable help and support!!!
